wfe/context.go

```diff
@@ -13,19 +13,17 @@ import (
 )
 
 type requestEvent struct {
-	ID            string `json:",omitempty"`
-	RealIP        string `json:",omitempty"`
-	ClientAddr    string `json:",omitempty"`
-	Endpoint      string `json:",omitempty"`
-	Method        string `json:",omitempty"`
-	Errors        []string
-	Requester     int64                  `json:",omitempty"`
-	Contacts      *[]string              `json:",omitempty"`
-	RequestNonce  string                 `json:",omitempty"`
```

Comment:
Thanks! Sure, removing it from the Nginx logs seems fine with me. I typically don't use those logs for debugging ACME problems and the duplication seems like a waste.